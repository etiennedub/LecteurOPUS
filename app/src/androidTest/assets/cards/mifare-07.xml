<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">0434BB03</List><List index="1">2A444981</List><List index="2">A648E000</List><List index="3">0000000F</List><List index="4">00000000</List><List index="5">30DF0523</List><List index="6">00000000</List><List index="7">FFC244F8</List><List index="8">01C2A73E</List><List index="9">73E60506</List><List index="10">19000000</List><List index="11">20281564</List><List index="12">01C2A5FC</List><List index="13">5FC60504</List><List index="14">A6008000</List><List index="15">2028E9C7</List></Pages></Mifare>