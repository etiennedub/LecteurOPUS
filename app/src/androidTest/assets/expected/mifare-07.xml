<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="28/09/2016" Time="15:27" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /><List index="1" Date="28/09/2016" Time="12:46" Transfert="false" OperatorId="20" BusId="96" OperatorName="RTC" BusName="METROBUS 801" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>